packages/nextjs/pages/index.tsx

```diff
@@ -48,76 +48,48 @@ const Home: NextPage<{
-            <ul className="list-disc list-inside flex flex-col space-y-3 m-auto lg:mx-0 max-w-[300px] lg:max-w-none">
-              <li>Learn quickly by building real apps</li>
-              <li>Speedrun the most important concepts</li>
-              <li>Level up your skills and get paid</li>
-            </ul>
-            <div className="text-center lg:text-left">
-              <TrackedLink
-                id="SpeedRunEthereum"
-                href="https://speedrunethereum.com/"
-                className="btn btn-accent btn-md lg:self-start px-8 hover:opacity-100"
-              >
-                Get Started Now!
-              </TrackedLink>
-            </div>
-          </div>
-          <div className="max-w-[300px] lg:max-w-none">
-            <Image src="/assets/sre-forest.png" alt="hero" width={500} height={500} />
-          </div>
-        </div>
-        <div className="bg-[url(/assets/sre-path.png)] bg-repeat-x h-32 relative bg-[35%_top]"></div>
-      </div>
-
-      {/* Test Your knowledge */}
-      <div className="bg-base-100 -mt-16">
-        <div className="container max-w-[90%] lg:max-w-6xl m-auto py-16 lg:py-28 lg:px-12">
-          <div className="bg-[url(/assets/dice.png)] bg-no-repeat bg-right-bottom lg:bg-right-top bg-[length:100px] lg:bg-auto max-w-[680px] pb-10">
+      {/* Start Building on Ethereum */}
+      <div className="bg-white pb-16" id="start-building-on-ethereum">
+        <div className="container max-w-[90%] lg:max-w-6xl m-auto py-16 lg:py-28 lg:px-12 flex flex-col items-center lg:items-start">
+          <div className="bg-[url(/assets/spaceship.png)] bg-no-repeat bg-right-top lg:max-w-[580px] bg-[length:85px] lg:bg-[length:100px] max-w-[500px] pb-10">
             <div className="mt-6 lg:mt-0">
               <h2 className="text-3xl lg:text-5xl text-center lg:text-left lg:max-w-lg">
-                Already have Solidity skills?
+                Start building <br /> on Ethereum
               </h2>
               <p className="text-center lg:text-left lg:w-3/4">
-                Test your knowledge by building real-world, complex apps.
+                Start your journey with Speedrun Ethereum, then continue learning in our Batches program and opt to our
+                small grants to improve your skills!
               </p>
             </div>
           </div>
           {/* Cards container */}
           <div className="flex gap-8 flex-wrap lg:flex-nowrap justify-center lg:max-w-5xl">
             {/* Card */}
             <Card
-              num={4}
-              name="Build a DEX"
+              name="Speedrun Ethereum"
               src="/assets/chall-dex.png"
-              description="💵 Build an exchange that swaps ETH to tokens and tokens to ETH."
+              description="Learn the most important concepts and level up your skills."
               link="https://github.com/scaffold-eth/se-2-challenges/tree/challenge-4-dex"
```

Comment:
We're still showing dex4 image and pointing to the dex 4 repo, is it ok?. Should we at least link to https://speedrunethereum.com/?

Reply:
Hey! Good point, we should definitely change the link to the SRE site, yes. 
About the image I'm not sure, I like that it shows a challenge title but let me get back to you on this one!

Reply:
Also, do you see this image like this? 
<img width="320" alt="Captura de pantalla 2024-09-30 a las 18 01 30" src="https://github.com/user-attachments/assets/592d67fe-19c3-4132-a118-ce95f1ff8601">

I thought I fixed that but now it's off again

Reply:
Nevermind, it's fixed now!